Multi-World Calendar/script.json

```diff
@@ -1,17 +1,16 @@
 {
-    "$schema": "https://github.com/Roll20/roll20-api-scripts/master/_Example%20Script%20-%20Check%20for%20formatting%20details/script.schema.json",
-    "name": "Multi-World Calendar",
-    "script": "calendar.js",
-    "version": "5.3",
-    "previousversions": ["5.2","5.1","5.0"],
```

Comment:
To avoid breaking changes for other users, I would just release a new version, and not remove existing ones.

Reply:
Fixed